src/content/docs/accounts/accounts/saml-single-sign/saml-service-providers.mdx

```diff
@@ -190,6 +190,21 @@ Here are some important procedures for managing SAML SSO for users on our [origi
 
 <CollapserGroup>
 
+  <Collapser
+    id="domain-allow-list"
+    title="Optional: Claim your domain to streamline SAML SSO process"
+  >
+
+Depending on your [subscription level](https://newrelic.com/pricing), you may have the option to put your domain name(s) on an allow list (whitelist), which will streamline the SAML SSO enablement process. When your Owner or Admins add new users and those users' email address has a domain that matches your claimed domain, New Relic automatically adds them as **Active** users and retains their previous user type. 
```

Comment:
Is there a doc we should be pointing them to instead? Or is the pricing page the best for info?